GPU/GPUTracking/DataTypes/GPUTRDTrack.h

```diff
@@ -131,6 +133,7 @@ class GPUTRDTrack_t : public T
 
  protected:
   float mChi2;                       // total chi2.
+  float mSignal{-1.f};               // electron Likelihood for track
```

Comment:
Please add `mSignal` also to the relevant methods in GPUTRDTrack.cxx. You can search for `mChi2` and add it everywhere where that is